contrib/codegen/templates/sets/sets.gotmpl

```diff
@@ -136,7 +137,7 @@ func (s *{{ $kindLowerCamel }}Set) Insert(
         {{ $kindLowerCamel }}List ...*{{ $import_prefix }}.{{ $resource.Kind }},
 ) {
     if s == nil {
-        panic("cannot insert into nil set")
+        contextutils.LoggerFrom(nil).DPanic("cannot insert into nil set")
```

Comment:
Same as above -- I think we can revert the changes in this file because it isn't problematic for codegen to panic